tests/pull-test.sh

```diff
@@ -35,7 +35,7 @@ function verify_initial_contents() {
     assert_file_has_content baz/cow '^moo$'
 }
 
-echo "1..27"
+echo "1..28"
```

Comment:
Can we also have an `ostree admin upgrade` test in `admin-test.sh` to validate the sysroot upgrader changes as well? (E.g. just checking that file objects were not fetched?).